README.md

```diff
@@ -1,3 +1,4 @@
 Welcome to GitHub
 
-Tharik Kanaka
\ No newline at end of file
+Tharik Kanaka
+Mohomed Rashad
```

Comment:
Can you create a directory/folder in your student no and create a read me there?